tests/openssl/enc/enc.c

```diff
@@ -68,7 +72,9 @@ int enc_test(int argc, char** argv, char** env)
      * framework such that we can free it (which the framework does not do
      * that). Without doing this, DEBUG_MALLOC will report memory leaks.
      */
+#if OECRYPTO_OPENSSL_VER < 3
```

Comment:
is there something else you need to do for openss 3? Does init_crypto cover this?

Reply:
This code line is to mitigate a memory leak in openssl 1.1's test infrastructure. (Specifically, test_open_streams and test_close_streams in test/testutil/basic_output.c.)

The BIO_f_tap API is removed in OpenSSL 3.0, so we need to remove this line to compile. The test infrastructure memory leak is fixed in OpenSSL 3.0, so we don't need to do anything to compensate.

This is unrelated to init_crypto. (The call to init_crypto should be moved to init.c once we have an oecryptolib that uses OpenSSL 3.0.)